docs/install-and-run.md

```diff
@@ -3,130 +3,164 @@
+#### Step 3: Open the workspace and verify that you see EarlGrey.
+
+   After you successfully run the `pod install` command, open the generated workspace and find EarlGrey
+   installed in the `Pods/` directory. The generated `Pods/` project should look similar to:
+
+   <img src="images/image02.png" width="250">
 
-EarlGrey supports Carthage via [the EarlGrey gem.](https://rubygems.org/gems/earlgrey)
+### Carthage Installation
 
 #### Step 1: Set up a test target for Carthage
 
-See [Step 1 from the `CocoaPods installation`](#step-1-set-up-a-test-target) detailed above.
+   See [Step 1 from the `CocoaPods installation`](#step-1:-set-up-a-unit-testing-target) detailed above.
 
 #### Step 2: Configure carthage
 
-Install Carthage
+   1. Install Carthage
+
+      For Carthage, you need Homebrew installed which you can get from [here](https://brew.sh/). We recommend keeping your version updated.
+
+      `brew update`
 
-`brew update; brew install carthage`
+      To install Carthage, execute:
 
-Specify the version of EarlGrey to use in Cartfile.private.
-Note that you can also use "master" instead of a release tag.
+      `brew install carthage`
 
-`echo 'github "google/EarlGrey" "1.2.0"' >> Cartfile.private`
+   2. Specify the version of EarlGrey to use in Cartfile.private.
 
-Update to latest EarlGrey revision and create Cartfile.resolved.
+      Note that you can also use "master" instead of a release tag.
 
-`carthage update EarlGrey --platform ios`
+      `echo 'github "google/EarlGrey" "1.2.0"' >> Cartfile.private`
+
+   3. Update to latest EarlGrey revision and create Cartfile.resolved.
+
+      `carthage update EarlGrey --platform ios`
 
 #### Step 3: Use the EarlGrey gem
 
-Install the EarlGrey gem.
+   1. Install the EarlGrey gem.
 
-`gem install earlgrey`
+      `gem install earlgrey`
 
-Use the gem to install EarlGrey into the testing target.
+   2. Use the gem to install EarlGrey into the testing target.
 
-`earlgrey install -t EarlGreyExampleSwiftTests`
+      `earlgrey install -t EarlGreyExampleSwiftTests`
 
-Now you're ready to start testing with EarlGrey!
-If you need more control, review the available install options.
+      Now you're ready to start testing with EarlGrey!
+      If you need more control, review the available installation options in the earlgrey gem:
 
-`earlgrey help install`
+      `earlgrey help install`
 
 ### GitHub Installation
 
-In cases where CocoaPods is not compatible with your project, you can add EarlGrey manually to your Xcode project.
+If CocoaPods or Carthage doesn't suit your use case, then you can add EarlGrey manually to your Xcode project.
 
 #### Step 1: Generate EarlGrey.framework
 
-  1. Download the source for EarlGrey and its dependencies.
-     * Download EarlGrey's source code from the [Latest Release](https://github.com/google/EarlGrey/archive/1.0.0.zip)
+   1. Download EarlGrey's source code from the
+      [Latest Release](https://github.com/google/EarlGrey/releases/tag/1.8.0).
```

Comment:
Lets link to https://github.com/google/EarlGrey/releases instead, the latest will be on the top any ways otherwise  we will have to keep updating this link